build-tools/packages/build-tools/src/common/fluidRepo.ts

```diff
@@ -321,46 +53,34 @@ export class FluidRepo {
 
 	public readonly packages: Packages;
 
-	public static create(resolvedRoot: string) {
-		const packageManifest = loadFluidBuildConfig(resolvedRoot);
-		return new FluidRepo(resolvedRoot, packageManifest);
-	}
-
-	protected constructor(
+	public constructor(
```

Comment:
why the change to public?

Reply:
Because we removed the FluidRepo.create and have caller directly do new FluidRepo.